app/furniture/marketplace/marketplaces/edit.html.erb

```diff
@@ -7,6 +7,10 @@
       <%= link_to t('marketplace.products.index.link_to'), marketplace.location(child: :products) %>
     <%- end %>
 
+    <%- if policy(marketplace.products.new).create? %>
```

Comment:
Is this the domain object you intended to be checkin?

Reply:
good catch. Thanks. fixed in 36c95afeec07768576932827639b0385d89328a5